packages/ui/src/theme/types/style-dictionary.d.ts

```diff
@@ -0,0 +1,26 @@
+declare module 'style-dictionary/lib/utils/references/usesReference.js' {
+  // based on the typing specified by style-dictionary:
+  // https://github.com/amzn/style-dictionary/blob/main/lib/utils/references/usesReference.js
+  export default function usesReference(value: string): boolean;
```

Comment:
Is there potential for drift on these types? I'm assuming these types cannot be exported from style-dictionary, is that correct?

Reply:
That is correct, something if I ever get time to do I would like to rewrite the whole thing in TS and expose the correct types with better exports so we don't have to import specific files. There are a lot of projects that do this so I promise not to break any of these paths :)